contributing/code_of_conduct/reporting_guidelines.rst

```diff
@@ -0,0 +1,98 @@
+Reporting Guidelines
+====================
+
+If you believe someone is violating the code of conduct we ask that you report
+it to the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+by emailing, twitter, in person or any way you see fit.
+
+**All reports will be kept confidential.** In some cases we may determine that
+a public statement will need to be made. If that's the case, the identities of
+all victims, reporters, and the accused will remain confidential unless those
+individuals instruct us otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate emergency
+services first by calling the relevant local authorities. If you are unsure what
+service or agency is appropriate to contact, include this in your report and we
+will attempt to notify them.
+
+In your report please include:
+
+  * Your contact info for follow-up contact.
+  * Names (legal, nicknames, or pseudonyms) of any individuals involved.
+  * If there were other witnesses besides you, please try to include them as well.
+  * When and where the incident occurred. Please be as specific as possible.
+  * Your account of what occurred.
+  * If there is a publicly available record (e.g. a mailing list archive or a
+    public IRC or Slack log), please include a link.
+  * Any extra context you believe existed for the incident.
+  * If you believe this incident is ongoing.
+  * Any other information you believe we should have.
+
+What happens after you file a report?
+-------------------------------------
+
+You will receive a reply from the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+acknowledging receipt as soon as possible, but within 24 hours.
+
+The team will immediately meet to review the incident and determine:
+
+  * What happened.
+  * Whether this event constitutes a code of conduct violation.
+  * What kind of response is appropriate.
+
+If this is determined to be an ongoing incident or a threat to physical safety,
+the team's immediate priority will be to protect everyone involved. This means
+we may delay an "official" response until we believe that the situation has ended
+and that everyone is physically safe.
+
+Once the team has a complete account of the events, they will make a decision as
+to how to respond. Responses may include:
+
+  * Nothing (if we determine no code of conduct violation occurred).
+  * A private reprimand from the Code of Conduct response team to the individual(s)
+    involved.
+  * A public reprimand.
```

Comment:
`* A public reprimand.`

Is public shaming really something you want to do and if so, why? It will, most likely, tarnish their reputation and, possibly, affect their future employment. This could be just as bad, if not worse, than reporting someone to their employer.

Reply:
yeah .. I also think this is something that isn't going to be done often .. if at all.

Reply:
This can also be illegal in some jurisdictions. As an example, Sweden (being my home jurisdiction and thus the one I am most familiar with) has the following law against defamation:

> Penal code
> 
> Chapter 5
> On Defamation
> 
> Section 1
> A person who points out someone as being a criminal or as having a reprehensible way of living or otherwise furnishes information intended to cause exposure to the disrespect of others, shall be
sentenced for defamation to a fine.
> If he was duty-bound to express himself or if, considering the circumstances, the furnishing of information on the matter was defensible, and if he can show that the information was true or that he had reasonable grounds for it, no punishment shall be imposed.
> 
> Section 2
> If the crime defined in Section 1 is regarded as gross, a fine or imprisonment for at most two years shall be imposed for gross defamation.
> In assessing whether the crime is gross, special consideration shall be given to whether the information, because of its content or the scope of its dissemination or otherwise, was calculated to bring about serious damage.

There is ample precedent that publicly shaming someone is considered defamatory, and the bar for it being defensible is fairly high. Furthermore, there's precedent that publishing a defamatory statement publicly on the internet constitutes gross defamation. Also, re-publishing a defamatory statement constitutes a separate offense, so if e.g. there were a post on the Symfony blog publicly reprimanding someone, and that entry is syndicated via RSS to say a Swedish tech news site, the news site could be liable.

Reply:
Magnus, defamation requires that what you say is false. For example I can say that the Swedish citizen [John Ingvar Lövgren](https://en.wikipedia.org/wiki/John_Ingvar_L%C3%B6vgren) is a criminal and a murderer. Given that this is a true fact, it's not defamation. I couldn't say the same about you, because that would be false and thus, defamation.

Reply:
Not in Sweden.

Reply:
Obviously, Wikipedia is not the best source, but I couldn't find any other english-language sources (other than the actual law, which I posted above, but that's not necessarily easily interpreted). [Here's what Wikipedia says](https://en.wikipedia.org/wiki/Defamation#Sweden):

> In Sweden, the criminal offense of denigration (ärekränkning) is regulated in Chapter 5 of the Criminal Code. Article 1 regulates defamation (förtal) and consists of pointing out someone as a criminal or as "having a reprehensible way of living", or of providing information about him "intended to cause exposure to the disrespect of others". The penalty is a fine. It is generally not a requirement that the statements are untrue, it is enough if they statements are meant to be vilifying.

Reply:
So Sweden is a curious exception then :)  By the way, what is the legal jurisdiction of the Symfony project? France?

Reply:
It's certainly an outlier, but I have no idea if it's the only outlier. As for the jurisdiction, who knows? I mean, is the project even it's own legal person?

Reply:
The brand "Symfony" is registered at least in France: https://bases-marques.inpi.fr/Typo3_INPI_Marques/marques_fiche_resultats.html?index=1&refId=4253977_201734_fmark&y=139
The company "Symfony" too: https://www.societe.com/societe/symfony-834643579.html
Maybe also in other countries, I don't know.

Reply:
well the question is if the enforcement team is not just a bunch of individuals. they will not be employed by Symfony/Sensio for working on the enforcement team. now of course someone from the enforcement team could be allowed by their employer to do this on work time. but IANAL .. but I would suspect the jurisdiction is going to be fuzzy and its going to be likely a question of where a case is filed, similar to anything else that is happening on the internet when individuals communicate.

Reply:
We need to double check this, because every action has (potential) legal consequences and we need to clarify everything. My guess is that team member actions fall under each person legal residence jurisdiction and abstract team actions under France jurisdiction.

Reply:
@javiereguiluz I don't think so. If there is no legal entity to represent, each member would probably be responsible for their own actions, and considered partly responsible (and likely culpable) for "abstract team actions", under their own separate jurisdictions.

Reply:
This bullet is removed.